pipelines/common/pom.xml

```diff
@@ -120,7 +120,11 @@
     <dependency>
       <groupId>ca.uhn.hapi.fhir</groupId>
       <artifactId>hapi-fhir-converter</artifactId>
```

Comment:
Now that you are doing these changes, can you please also check if this dependency is still needed? We used to convert R4 resources to STU3 and that's why we had this dependency, but we don't do that anymore.